### DISCUSSION TITLE
get text message in InlineMenu

### DISCUSSION THREAD

**Discussion by ghost** [nutgram/nutgram#623] [Q&A] [answered]
Hi.
How can I get user text somewhere in my Inline menu instead of button click. And I need to return to Inline menu step just after this message. 

For example -- 
How to get 'wow' as text in function 'none' in this example
https://nutgram.dev/docs/usage/inline_menu

Thank you.

**Comment by Lukasss93** [comment, marked as answer]
```php
public function none(Nutgram $bot)
{
    $value = $bot->message()->text; // <--
	
    $bot->sendMessage('Bye!');
    $this->end();
}
```

**Reply by ghost** [reply]
Thank you!
